# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../betaftpd/hotfix.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./cmds.h.orig	2008-08-25 17:55:12.000000000 +0000
+++ ./cmds.h	2008-08-25 18:02:51.000000000 +0000
@@ -54,7 +54,7 @@
 	}
 #endif
 
-#define CMD_PROTO(cmd) int cmd_ ## cmd ## (struct conn * const c)
+#define CMD_PROTO(cmd) int cmd_ ## cmd (struct conn * const c)
 
 int do_chdir(struct conn * const c, const char * const newd);
 CMD_PROTO(user);
